repl/src/test/scala/com/cloudera/livy/repl/Python3InterpreterSpec.scala

```diff
@@ -208,15 +208,5 @@ class Python3InterpreterSpec extends BaseInterpreterSpec {
     ))
   }
 
-  // Scalastyle is treating unicode escape as non ascii characters. Turn off the check.
-  // scalastyle:off non.ascii.character.disallowed
-  it should "print unicode correctly" in withInterpreter { intp =>
-    intp.execute("print(u\"\u263A\")") should equal(Interpreter.ExecuteSuccess(
```

Comment:
Sorry for the late reply.
Feel free to remove case "\xE2\x98\xBA". But please keep other test cases.

Also, I think we can use io.StringIO() directly for stdout and stderr in Python3:
Mind if u try this and see does it work?

```python
if sys.version >= '3':
  sys.stdin = io.StringIO()
  sys.stdout = io.StringIO()
  sys.stderr = io.StringIO()
else:
  sys.stdin = cStringIO.StringIO()
  sys.stdout = UnicodeDecodingStringIO()
  sys.stderr = UnicodeDecodingStringIO()
```